app/src/main/java/org/oppia/app/profile/ProfileChooserViewModel.kt

```diff
@@ -1,5 +1,6 @@
 package org.oppia.app.profile
 
+import androidx.fragment.app.Fragment
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.Transformations
 import org.oppia.app.fragment.FragmentScope
```

Comment:
Also instead of `import java.util.*` use import `import java.util.Locale`

Reply:
Done